pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -110,9 +98,43 @@ workflow RNAWithUMIsPipeline {
       read2Structure = read2Structure
   }
 
-  call tasks.STAR {
+  # Convert SAM to fastq for adapter clipping
+  # This step also removes reads that fail platform/vendor quality checks
+  call tasks.SamToFastq {
     input:
       bam = ExtractUMIs.bam_umis_extracted,
+      output_prefix = output_basename
+  }
+
+  # Adapter clipping
+  call tasks.Fastp {
+    input:
+      fastq1 = SamToFastq.fastq1,
+      fastq2 = SamToFastq.fastq1,
```

Comment:
Bug!
```suggestion
      fastq2 = SamToFastq.fastq2,
```

Reply:
@takutosato I fixed this - I'm 99.999999% certain it was a bug, but please verify